Main page collection

oksana12345678 opened the pull request:
✅ Added DotsSwiper component for tablet devices to display collections in a slider.
✅ Integrated ProgressBar component to visually represent collection progress.
✅ Added ActiveCollectionsCard component to render individual collection details.
✅ Implemented ActiveCollectionList component to render ActiveCollectionsCard on mobile & desktop devices.
✅ Created collections.json to store and use collection data dynamically.
✅ Styled all components using SCSS modules for a consistent and responsive UI.

z9877969 commented on src/shared/components/ProgressBar/ProgressBar.jsx:
даний компонент краще обізвати `CollectionProgress` і перенести в компонети модуля `mainPageCollection`, а `ProgressBar` реалізувати як компонент, що рендерить лише сам  **bar** - полосу

z9877969 commented:
загалом хороша робота.
трохи накидав зауважень, стосовно поверхневого рефакторингу коду.
пул-реквест тимчасово закриваю, коли пофіксиш, відкриєш його ж ще раз.

oksana12345678 commented:
-fixed all remarks from comments
-added re-export to all components 
-divide ProgressBar component to ProgressBar'; & CollectionProgress
-changed font-size from .scss to _mixins font
